web/libs/core/src/blocks/TokenSettingsModal/index.tsx

```diff
@@ -59,6 +56,10 @@ export const TokenSettingsModal = ({
           />
         </Form.Row>
       )}
+      <Form.Actions>
+        <Form.Indicator />
```

Comment:
oh, great! we actually don't need to pass children in it, I didn't know, will fix some settings then